index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+
+<title>Emelio's Site</title>
+<link rel="stylesheet" href="style.css"/>
+
+
+<header>
+  <h1 id="header"> Emilio Comiling</h1>
+  <img class= "image" src="emelio.jpg">
```

Comment:
Naming conventions. It's best practice to use consistent naming conventions through a project. Use names that explain exactly what the element is. As a project grows, it will benefit having more specific class/id names. 
```suggestion
  <img class= "header-image" src="emelio.jpg">
```